usr/share/rear/lib/serial-functions.sh

```diff
@@ -0,0 +1,42 @@
+# Get available serial devices
+function get_serial_devices {
+    if [ -z $SERIAL_CONSOLE_DEVICES ]; then
+        serial_devices=$(ls /dev/ttyS[0-9]* /dev/hvsi[0-9]* | sort)
+        echo "$serial_devices"
+    else
+        $SERIAL_CONSOLE_DEVICES
+    fi
+}
+
+function build_cmdline {
+    # Enable serial console, unless explicitly disabled
+    if [[ ! "$USE_SERIAL_CONSOLE" =~ ^[yY1] ]]; then
+        return
+    fi
+
+    # Add serial console to /etc/inittab and kernel cmdline
+    # ignore console kernel cmd parameters
+    cmdline=
+    for param in $KERNEL_CMDLINE; do
+        case "$param" in
+            (console=*) ;;
+            (*) cmdline="$cmdline$param ";;
+        esac
+    done
+
+    # add seril config to kernel cmd line
```

Comment:
Typo "seril" should be "serial" but "serial" is not what it actually is about.
Actually it is about to specify the "console" kernel command line option
so we should use the word "console" when we do things related to it.